react/components/atoms/text-input/text-input.js

```diff
@@ -0,0 +1,90 @@
+import React, { PureComponent } from "react";
+import { StyleSheet, Text, TextInput, View } from "react-native";
+import PropTypes from "prop-types";
+import { mix } from "yonius";
+
+import { IdentifiableMixin, baseStyles } from "../../../util";
+
+export class TextInputRipe extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            title: PropTypes.string,

```

Comment:
maybe call it label, which is something we normally call this in web. If not label, at least header, to make it more similar to components-vue's `form-input`